popup.js

```diff
@@ -360,11 +452,67 @@ document.addEventListener("DOMContentLoaded", function () {
     });
   }
 
-  function logout() {
```

Comment:
Can you just undo these changes for logout button. It would be quick you just to copy paste this code back.